app/classes/actions.ts

```diff
@@ -2,22 +2,24 @@
 
 import { initTransport } from "@/lib/transport"
 import { TwirpError } from "twirpscript"
-
 import { authHeader } from "@/app/actions"
 import {
   UpcomingSessionResponse,
   UpcomingSessions,
   UpcomingSessionsRequest,
 } from "@/lib/fusion/masterClass/session.manager.pb"
 import { cookies } from "next/headers"
+import { GetMasterClass, GetMasterClassRequest } from "@/lib/fusion/masterClass/masterClass.repository.sky.pb"
+import { MasterClass } from "@/lib/fusion/masterClass/masterClass.pb"
 
 initTransport()
 
 export async function upcomingSessions(
   request: UpcomingSessionsRequest,
 ): Promise<UpcomingSessionResponse> {
   try {
-    cookies().getAll() // read cookies to prevent cache
+    // TODO: Why was cookies gotten here? Why did we want to prevent cache?
```

Comment:
Because otherwise we show session in productions momentarily even though they aren't available anymore. The caching is not enabled when running in local development mode

Reply:
The problem here is that the request needs to be made at build time in order to statically generate the individual master class page. The cookies aren't available at build time.

It doesn't seem right that the data for the page should be dependent on the cookies. 

Claude thinks we should rather use `Server Side Rendering` instead of `Static Site Generation`.
But I want to use static generation for those pages as they don't need to be rehydrated.

We could do cache prevention on a specific path by setting a cache control header in the next.js config.
https://nextjs.org/docs/pages/api-reference/next-config-js/headers#cache-control